playbooks/cabinet-office/rocketchat.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+- name: Bootstrap instances
+  hosts: all
+  gather_facts: no
+  become: true
+  roles:
+    - python
+
+- name: Install Rocket Chat service
+  hosts: rocketchat_servers
+  gather_facts: yes
+  become: true
+  roles:
+    - role: nginx
+      nginx_extra_sites:
+        - "{{ ROCKETCHAT_NGINX_SITE_CONFIG }}"
+    - role: RocketChat.Server
+  tasks: # This is valid for Ubuntu 16.04
+    - name: Deploy the Rocket.Chat service file override
```

Comment:
What about adding a "when" statement here to run this only if rocket_chat_service_template_override variable is defined? ammm well I think it's covered by the ignore_errors flag. I leave it to you: keep it as it is or using the when